src/device-registry/controllers/create-event.js

```diff
@@ -134,7 +134,7 @@ const createEvent = {
         if (format && format === "csv") {
           return res
             .status(status)
-            .type("text/csv")
+            .type("text/html")
```

Comment:
disable automatic Browwer download of CSV files for device measurements whenever` format=csv`